packages/tasks/src/model-libraries.ts

```diff
@@ -172,6 +172,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/fudan-generative-vision/champ",
 		countDownloads: `path:"champ/motion_module.pth"`,
 	},
+	chatterbox: {
+		prettyLabel: "Chatterbox",
+		repoName: "Chatterbox",
+		repoUrl: "https://github.com/resemble-ai/chatterbox",
+		snippets: snippets.chatterbox,
+		countDownloads: `path:"tokenizer.config"`,
```

Comment:
```suggestion
		countDownloads: `path:"tokenizer.json"`,
```

the file in the repo seems to be named .json: https://huggingface.co/ResembleAI/chatterbox/tree/main